eng/restore/harvestPackages.targets

```diff
@@ -23,6 +23,7 @@
 
     <!-- Allow to override package download and versions in case there is already a PackageDownload set -->
     <ItemGroup>
+      <PackageDownload Include="System.Drawing.Common" Version="4.5.2" />
```

Comment:
We can now remove this, we can now harvest from 3.1 package since it is already in nuget. (4.7.2, which should be taken from the packageindex.json)